csharp/Yoga/resource.h

```diff
@@ -0,0 +1,14 @@
+//{{NO_DEPENDENCIES}}
```

Comment:
What are Yoga.rc and resource.h used for?

---
In reply to: [92471469](https://github.com/facebook/yoga/pull/283#discussion_r92471469) [](ancestors = 92471469)